MTM-44133 related topics for device management (#546)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/connecting-devices.md

    **Accepted** - the user has allowed the credentials to be send to the device"
---

{{< c8y-admon-related >}}

> * See [Device integration tutorials](/device-tutorials/tutorials-introduction) for information on the integration of specific devices

Reviewer: @rssag @MWindel We need to decide on various format questions and unify accordingly. 
I always use a full stop at the end.
I do not use "See" at the beginning.

Author: added full stops, removed "see", and done @simple demo devices.

* See the [Device SDK guide](/device-sdk/introduction/) for information on device integration via REST and C++

> * See the [device control API](https://cumulocity.com/api/{{< c8y-current-version >}}/#tag/Device-control-API) for REST API methods concerning device management

Reviewer: Perhaps add one more: See the documentation for the protocols in the Protocol integration guide for registering devices of a specific type?

Author: added the entry

    **Accepted** - the user has allowed the credentials to be send to the device"
---

{{< c8y-admon-related >}}

> * [Device integration tutorials](/device-tutorials/tutorials-introduction) for information on the integration of simple demo devices.

Reviewer: ```suggestion
* [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by step instructions on registering simple demo devices.
```

> * [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by step instructions on registering simple demo devices.

Reviewer: ```suggestion
* [Device integration tutorials](/device-tutorials/tutorials-introduction) for step-by-step instructions on registering simple demo devices.
```

> * The [Device SDK guide](/device-sdk/introduction/) for information on device integration via REST and C++.

Reviewer: ```suggestion
* The [Device SDK guide](/device-sdk/introduction/) for detailed descriptions on device integration via REST and C++.
```

> * The [device control API](https://cumulocity.com/api/{{< c8y-current-version >}}/#tag/Device-control-API) for REST API methods concerning device management.

Reviewer: concerning device management > is this only one API for the entire device management?

Author: no, but "device management" in this place is too generic.
i specified the path to "New device request API" and changed the text accordingly.

    To register a device, click **Register device** at the right of the top bar, select an option from the dropdown list and follow the instructions in the device registration wizard."
---

{{< c8y-admon-related >}}
* [Device integration > Integration tutorials](/device-integration/integration-tutorials) for step-by-step instructions on registering devices.
* [Device integration using REST](/device-integration/rest/) for a detailed descriptions on device integration via REST.

> * The [New device requests API](https://cumulocity.com/api/core/{{< c8y-current-version >}}/#tag/New-device-requests) for REST API methods concerning the creation of new devices.

Reviewer 2: ```suggestion
* The [New device requests API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/New-device-requests) for REST API methods concerning the creation of new devices.
```

> * The [Protocol integration guide](/protocol-integration/overview) for details on registering devices using various standard protocol types.

Author: made it the first entry
